packages/cli/templates/platform-app-tailwind/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "appId": "<Your app ID>",
+    "appType": "platform-app",
+    "entryPoint": [
+        {
+            "type": "ASSET_CREATION"
+        },
+        {
+            "type": "ASSET_ACTION"
+        }
```

Comment:
Perhaps for the template, we could only have a single entry point, in order not to overcomplicate things?

Also, just so we wouldn't forget, there is [a follow-up task](https://app.clickup.com/t/8692hdwh5) for web-app, once we're ready to switch it from string to array (currently, the `custom_fields` value is `'{\"source\": \"https://remove-background-action.vercel.app/\", \"entryPoint\": \"ASSET_ACTION\"}'`)